src/actions/config.js

```diff
@@ -9,6 +9,11 @@ export const CONFIG_SUCCESS = "CONFIG_SUCCESS";
 export const CONFIG_FAILURE = "CONFIG_FAILURE";
 export const CONFIG_MERGE = "CONFIG_MERGE";
 
+const validTypes = { 'text/yaml': 'yaml', 'application/x-yaml': 'yaml', 'application/toml': 'toml' };
+const configLink = document.querySelector('link[rel="cms-config-url"]');
+const isValidType = link => link && validTypes[link.type]; 
+const configUrl = isValidType(configLink) ? get(configLink, 'href') : 'config.yml';
```

Comment:
It'd be nice to group all of the logic into a `getConfigUrl` function.

Reply:
You got it!